jmeter-java-dsl/src/test/java/us/abstracta/jmeter/javadsl/core/BuildTreeContextTest.java

```diff
@@ -0,0 +1,55 @@
+package us.abstracta.jmeter.javadsl.core;
+
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.*;
+
+import java.io.File;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.LinkedHashMap;
+
+public class BuildTreeContextTest {
+
+    @Test
+    public void setEntryAddsKeyValuePair() {
+        BuildTreeContext context = new BuildTreeContext();
+        String key = "key";
+        String value = "value";
+
+        context.setEntry(key, value);
+
+        assertEquals(value, context.getEntry(key));
```

Comment:
we are using assertj for assertions, can you change it to follow the same convention?